wazo_websocketd/bus.py

```diff
@@ -380,7 +373,7 @@ async def _stop_consuming(self) -> None:
         self._connection.remove_consumer(self)
 
     async def bind(self, event_name: str) -> None:
-        bindings = [{}]
+        bindings = [{'origin_uuid': self._origin_uuid}]
```

Comment:
Could make the code more explicit that bindings is only for user and not for admin (using if/else) statement

Reply:
Refactored code to (hopefully) make it easier / more explicit